api-all.js

```diff
@@ -11,14 +11,14 @@ module.exports = function api( options ) {
   })
 
   this.add( 'role:api,path:shop', function( msg, respond ) {
-    var shopmsg = { role:'shop', id:msg.id }
+    var shopmsg = { role:'shop', id:msg.pid }
```

Comment:
Changed to use pid due to issue https://github.com/senecajs/seneca/issues/384 in seneca entity.